torch_tvm/compiler.cpp

```diff
@@ -23,24 +28,33 @@ tvm::relay::Var TVMCompiler::convertToRelay(const Value* val) {
 }
 
 tvm::relay::Expr TVMCompiler::convertToRelay(const IValue& val) {
+  // All doubles are converted to floats
```

Comment:
looks like we can convert to `float64` for double, can you add a comment here why the conversion to `float32` is necessary?

Reply:
it's not really necessary, I'll swap to fp64